### ISSUE TITLE
Inconsistency in syntax between `savecube` and `savedataset`

### ISSUE THREAD

**Issue by Qfl3x** [JuliaDataCubes/YAXArrays.jl#345]
Why is the path `savecube` a "normal" argument, while path in `savedataset` a kwarg?

**Comment by lazarusA** [comment]
Indeed. Things should be unified. There is an open issue about it as well: https://github.com/JuliaDataCubes/YAXArrays.jl/issues/309. Some work needs to be done also here.

**Comment by Qfl3x** [comment]
Another thing is that `path` isn't a mandatory kwarg, and it has a default value of `""`, wouldn't that create unexpected behavior?
